common/persistence/elasticsearch/esVisibilityMetricClients.go

```diff
@@ -55,14 +55,17 @@ func (p *visibilityMetricsClient) RecordWorkflowExecutionStarted(
 	ctx context.Context,
 	request *p.RecordWorkflowExecutionStartedRequest,
 ) error {
-	p.metricClient.IncCounter(metrics.ElasticsearchRecordWorkflowExecutionStartedScope, metrics.ElasticsearchRequests)
+	
+	scopeWithDomainTag := p.metricClient.Scope(metrics.ElasticsearchRecordWorkflowExecutionStartedScope,metrics.DomainTag(request.Domain))
+	scopeWithDomainTag.IncCounter(metrics.ElasticsearchRequests)
+	
+	sw := scopeWithDomainTag.StartTimer(metrics.ElasticsearchLatency)
 
-	sw := p.metricClient.StartTimer(metrics.ElasticsearchRecordWorkflowExecutionStartedScope, metrics.ElasticsearchLatency)
 	err := p.persistence.RecordWorkflowExecutionStarted(ctx, request)
 	sw.Stop()
 
 	if err != nil {
-		p.updateErrorMetric(metrics.ElasticsearchRecordWorkflowExecutionStartedScope, err)
+		p.updateErrorMetric(scopeWithDomainTag,metrics.ElasticsearchRecordWorkflowExecutionStartedScope, err)
```

Comment:
nit: need a space after `,`. Could you run `make lint` to have the linter fix this?